src/scanoss/scanner.py

```
@@ -52,12 +53,13 @@
 FAST_WINNOWING = False
 try:
     from scanoss_winnowing.winnowing import Winnowing
+
     FAST_WINNOWING = True
 except ModuleNotFoundError or ImportError:
```

Ruff (0.8.2)

Exception to catch is the result of a binary `or` operation

(PLW0711)